src/System.Diagnostics.TraceSource/src/System/Diagnostics/DefaultTraceListener.cs

```diff
@@ -20,7 +20,9 @@ namespace System.Diagnostics
     public class DefaultTraceListener : TraceListener
     {
         private const int internalWriteSize = 16384;
```

Comment:
Nit: I realize this preexists your change, but internalWriteSize should be InternalWriteSize